File: lesson7giang/src/main/java/com/example/lesson7giang/Student.java
package com.example.lesson7giang;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "student_table")
public class Student {
    @PrimaryKey
    @NonNull
    @ColumnInfo(name = "studentName")
    private String mName;

    @NonNull
    @ColumnInfo(name = "studentPhone")
    private String mPhone;

    @ColumnInfo(name = "studentAge")
    private int mAge;

    public Student(@NonNull String mName, @NonNull String mPhone, int mAge) {
        this.mName = mName;
        this.mPhone = mPhone;
        this.mAge = mAge;
    }

    @NonNull
    public String getName() {
        return this.mName;
    }

    @NonNull
    public String getPhone() {
        return this.mPhone;
    }

    public int getAge() {
        return this.mAge;
    }
}

File: lesson7khanh/src/main/java/com/k001/lesson7khanh/Student.kt
package com.k001.lesson7khanh

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "student_table")
data class Student(@PrimaryKey
                   @ColumnInfo(name = "student_name") val name: String,
                   @ColumnInfo(name = "student_age") val age: String,
                   @ColumnInfo(name = "student_telephone") val telephone: String)